2021-11-20 20:28:24,819 INFO    MainThread:12040 [wandb_setup.py:_flush():71] setting env: {}
2021-11-20 20:28:24,819 INFO    MainThread:12040 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-20 20:28:24,819 INFO    MainThread:12040 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-20 20:28:24,819 INFO    MainThread:12040 [wandb_init.py:_log_setup():371] Logging user logs to d:\Documents\Academics\Fall 2021\TCSS551\Project-1-Titanic-Classification\wandb\run-20211120_202824-2a9ens9j\logs\debug.log
2021-11-20 20:28:24,819 INFO    MainThread:12040 [wandb_init.py:_log_setup():372] Logging internal logs to d:\Documents\Academics\Fall 2021\TCSS551\Project-1-Titanic-Classification\wandb\run-20211120_202824-2a9ens9j\logs\debug-internal.log
2021-11-20 20:28:24,820 INFO    MainThread:12040 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000204F3182DD0>
2021-11-20 20:28:24,820 INFO    MainThread:12040 [wandb_init.py:init():404] calling init triggers
2021-11-20 20:28:24,820 INFO    MainThread:12040 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-20 20:28:24,820 INFO    MainThread:12040 [wandb_init.py:init():449] starting backend
2021-11-20 20:28:24,820 INFO    MainThread:12040 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-11-20 20:28:24,823 INFO    MainThread:12040 [backend.py:ensure_launched():199] starting backend process...
2021-11-20 20:28:24,896 INFO    MainThread:12040 [backend.py:ensure_launched():204] started backend process with pid: 31224
2021-11-20 20:28:24,896 INFO    MainThread:12040 [wandb_init.py:init():458] backend started and connected
2021-11-20 20:28:24,898 INFO    MainThread:12040 [wandb_run.py:_label_probe_notebook():917] probe notebook
2021-11-20 20:28:24,898 INFO    MainThread:12040 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-11-20 20:28:24,898 INFO    MainThread:12040 [wandb_init.py:init():519] updated telemetry
2021-11-20 20:28:24,930 INFO    MainThread:12040 [wandb_init.py:init():549] communicating current version
2021-11-20 20:28:26,380 INFO    MainThread:12040 [wandb_init.py:init():554] got version response 
2021-11-20 20:28:26,380 INFO    MainThread:12040 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-20 20:28:26,551 INFO    MainThread:12040 [wandb_init.py:init():592] starting run threads in backend
2021-11-20 20:28:28,325 INFO    MainThread:12040 [wandb_run.py:_console_start():1816] atexit reg
2021-11-20 20:28:28,326 INFO    MainThread:12040 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-11-20 20:28:28,326 INFO    MainThread:12040 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-11-20 20:28:28,327 INFO    MainThread:12040 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-20 20:28:28,327 INFO    MainThread:12040 [wandb_init.py:init():619] run started, returning control to user process
2021-11-20 20:28:28,329 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,329 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,352 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,353 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,354 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,369 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,382 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,382 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,401 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,409 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,409 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,425 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,427 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,427 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,442 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,449 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,449 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:28,468 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:28,469 INFO    MainThread:12040 [wandb_init.py:_pause_backend():284] pausing backend
2021-11-20 20:28:28,469 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:33,663 INFO    MainThread:12040 [wandb_init.py:_resume_backend():293] resuming backend
2021-11-20 20:28:33,670 INFO    MainThread:12040 [wandb_run.py:finish():1463] finishing run nclgbd/Titanic-Classification/2a9ens9j
2021-11-20 20:28:33,671 ERROR   MainThread:12040 [jupyter.py:save_history():421] Run pip install nbformat to save notebook history
2021-11-20 20:28:33,671 INFO    MainThread:12040 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2021-11-20 20:28:33,671 INFO    MainThread:12040 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2021-11-20 20:28:33,671 INFO    MainThread:12040 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-20 20:28:33,671 INFO    MainThread:12040 [wandb_run.py:_restore():1758] restore
2021-11-20 20:28:34,342 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1219
  total_bytes: 1219
}

2021-11-20 20:28:34,631 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1219
  total_bytes: 1219
}

2021-11-20 20:28:35,582 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1219
  total_bytes: 7790
}

2021-11-20 20:28:35,684 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1219
  total_bytes: 7790
}

2021-11-20 20:28:35,785 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5397
  total_bytes: 7790
}

2021-11-20 20:28:35,886 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:35,987 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:36,088 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:36,190 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:36,291 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:36,500 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}

2021-11-20 20:28:36,837 INFO    MainThread:12040 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7790
  total_bytes: 7790
}
local_info {
}

2021-11-20 20:28:37,959 INFO    MainThread:12040 [wandb_run.py:_append_files():2184] logging synced files
